documentation/aws-account-and-identity-architecture.md

```diff
@@ -105,6 +117,45 @@ stored in:
 
 The DevOps team recommends that project teams adopt automated continuous delivery to deploy and configure applications.
 
+
+**Identities**
+
+Each project account will have a limited set of IAM roles for people and services. Helpful Engineers will use the
```

Comment:
```suggestion
Each project account will have a limited set of IAM roles for people and services. Helpful Engineering will use the
```

Why are we using the term `engineers` instead of `engineering`?

Reply:
This might be a bit subtle, but I was trying to communicate connect the people of HE to their roles by using the noun Engineer.  Will drop to lower case.